tensorflow_quantum/core/ops/math_ops/tfq_inner_product.cc

```diff
@@ -182,12 +182,6 @@ class TfqInnerProductOp : public tensorflow::OpKernel {
         qsim::ApplyFusedGate(sim, fused_circuits[i][j], sv);
       }
       for (int j = 0; j < other_fused_circuits[i].size(); j++) {
-        // (#679) Just ignore empty program
-        if (fused_circuits[i].size() == 0) {
-          (*output_tensor)(i, j) = std::complex<float>(1, 0);
-          continue;
-        }
-
```

Comment:
Does removing this change the empty case behavior for the forward pass ?

Reply:
I thought that the configuration that the one program has cirq.X(q) and other_program is empty is possible (so, the answer should be 0.0 for <1|0>, not 1.0), but I got this error when I tried:

```
tensorflow.python.framework.errors_impl.InvalidArgumentError: A reference circuit contains qubits not found in paired circuit. [Op:TfqInnerProduct]
```

So, one is empty, the other should be empty. I missed it before. Please let me roll it back. thanks.